src/WatchDogs.FakeSource/IDataGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace WatchDogs.FakeSource;
+
+public interface IDataGenerator
```

Comment:
What about IFakeTradeGenerator? IDataGenerator sounds generic but it is only generating trades.

Reply:
ok.